include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -2477,47 +3160,1289 @@ namespace internal
     }
   };
 
-
-
-  template <int dim, int fe_degree, typename Number, bool lex_faces = false>
-  struct FEFaceNormalEvaluationImpl
+  template <bool symmetric_evaluate,
+            int  dim,
+            int  fe_degree,
+            int  n_q_points_1d,
+            typename Number>
+  struct FEFaceEvaluationImplRaviartThomas
   {
-    template <bool do_evaluate, bool add_into_output>
-    static void
-    interpolate(const unsigned int                            n_components,
-                const EvaluationFlags::EvaluationFlags        flags,
-                const MatrixFreeFunctions::ShapeInfo<Number> &shape_info,
-                const Number *                                input,
-                Number *                                      output,
-                const unsigned int                            face_no)
+    // We enable a transformation to collocation for derivatives if it gives
+    // correct results (first two conditions), if it is the most efficient
+    // choice in terms of operation counts (third condition) and if we were
+    // able to initialize the fields in shape_info.templates.h from the
+    // polynomials (fourth condition).
+    static_assert(
+      !symmetric_evaluate,
+      "symmetric_evaluate = true is not yet implemented for Raviart-Thomas");
```

Comment:
At this stage, I would systematically leave out the symmetric evaluate path here and in all functions below. My point is that we need to look things through at that stage anyway, so I see no harm in keeping the code smaller for now and avoid some TODOs.